### DISCUSSION TITLE
crystal shape，The orientation of the cell is wrong  mtex 5.11.1

### DISCUSSION THREAD

**Discussion by xiu-shen** [mtex-toolbox/mtex#2107] [Ask Anything] [answered]
I want to map the cell of a specific grain and the slip system

**Comment by ralfHielscher** [comment]
Hi Xiu,

- option 1 follows your default plotting convention, i.e. what you have specified with  `plotx2east` ....
- option 2 does not follow any plotting convention
- you can make option 2 following the default plotting convention by the command `setCamera('default')`

This will resolve you issue.

Ralf.

**Reply by xiu-shen** [reply]
Thank you for your reply. After I added setCamera('default'), there was no change in the orientation of the cell

**Comment by ralfHielscher** [comment, marked as answer]
For me the following code works perfectly
```matlab
figure(1)
grains =calcGrains(ebsd('indexed'))
cS = crystalShape.hex(ebsd.CS);
plot(grains.boundary)
hold on
plot(grains, cS * 0.8,'linewidth', 1, 'colored')
hold off

[x, y] = ginput(1);
ori = ebsd(x,y).orientations
figure(2);
plot(ori * cS,'colored')
setCamera('default')
```

**Reply by xiu-shen** [reply]
Thank you very much. My problem has been solved

**Comment by xiu-shen** [comment]


**Reply by kilir** [reply]
a) you can move the crystal shape accordingly e.g.
```Matlab
cs = crystalSymmetry('622')
cS = crystalShape.hex(cs)

plot(cS+vector3d(0,0,max(cS.V.z)),'faceAlpha',0.5)
hold on
iF = Miller(0,0,1,cs)
plotInnerFace(cS,iF,'faceColor','r');
hold off
```

Or simply select the plane of the shape (if there a corresponding one) and colorize this one.

Hope this helps.

Cheers,
Rüdiger

**Reply by xiu-shen** [reply]
Thank you for your help
